runtimes/eden/src/pallets_util.rs

```diff
@@ -138,3 +139,47 @@ impl pallet_uniques::Config for Runtime {
 	type CreateOrigin = AsEnsureOriginWithArg<EnsureSigned<AccountId>>;
 	type Locker = ();
 }
+
+parameter_types! {
+	pub const DepositPerItem: Balance = constants::deposit(1, 0);
+	pub const DepositPerByte: Balance = constants::deposit(0, 1);
```

Comment:
So what is this in NODL?

Reply:
It goes to our pre-existing function deposit where we have had specified how much per item and per byte should be deposited:
```
pub const fn deposit(items: u32, bytes: u32) -> Balance {
	items as Balance * 1_500 * MICRO_NODL + (bytes as Balance) * 600 * MICRO_NODL
}
```

Reply:
So `DepositPerItem` is `1_500 * MICRO_NODL` and `DepositPerByte` is `600 * MICRO_NODL`. Any reasons for not making it explicit?

Reply:
The `deposit` function is reused for configuring pallet_multisig and pallet_preimage and potentially more in the future. It's for the sake of consistency that we follow the same pricing scheme everywhere.

Reply:
Makes sense